api_mt/Trados.md

```diff
@@ -0,0 +1,121 @@
+## 2. How to set up Trados Studio 2021™ plugins
+### 2.1 Amazon Translate (AWS)
+#### 2.1.1 How to obtain an API key
+Refer to [Point 4.1](#41-amazon-translate-aws) for details.
+
+#### 2.1.2 How to set up the Trados Studio 2021™ plugin for Amazon Translate (AWS)
+ - Open your desktop version of Trados Studio 2021™.
+ 
+ - Go to **Add-ins > RWS AppStore**. Search for **Amazon Translate MT Provider** and download it. Restart Trados Studio 2021™.
+ 
+ - Go to **Project Settings**. In the **Translation Memory and Automated Translation** settings window select the provider by clicking **Use...** > **Amazon Translate Trados Plugin**.
+
+<!-- <img src="https://picc.io/_iBPzvh.png" alt="How to set up the Trados Studio 2021™ plugin for Amazon Translate (AWS)_pic1" style="height: 400px; width: 800px;"/> -->
+![How to set up the Trados Studio 2021™ plugin for Amazon Translate (AWS)pic1](./images/trados/trados_AWS/2021-10-11_21_33_42-5.png)
+
+<!-- <img src="https://picc.io/be6e_v7.png" alt="How to set up the Trados Studio 2021™ plugin for Amazon Translate (AWS)_pic2" style="height: 200px; width: 800px;"/> -->
+![How to set up the Trados Studio 2021™ plugin for Amazon Translate (AWS)pic2](./images/trados/trados_AWS/2021-10-11_21_34_52-6.png)
+
+- Enter necessary credentials in the **Authentication** window.
+
+    - Select **Access key  / Secret access key** in the **Choose AWS auth type** drop-down menu.
+		
+    - Enter the **AWS region name**, **Access key**, and **Secret access key** credentials in the appropriate fields. Check the **Save access keys for future sessions** option.
+    
+     - Confirm by clicking **OK**.
+
+![How to set up the Trados Studio 2021™ plugin for Amazon Translate (AWS)pic3](./images/trados/trados_AWS/2021-10-11_21_35_35-7.png)
```

Comment:
Let's use cleaner image names and keep images next to the articles they're used in, if possible.

e.g. `integrations/trados/amazon-translate/amazon-translate-trados-studio-2021-plugin.png`

Reply:
Sure. I shall change it (If not today then ASAP). Just one question: what do you mean by "next to the articles"? A special folder?

Reply:
Yes, but always keep folder and file names lowercase (so memoq not memoQ).